src/SdkCommon/AppAuthentication/Azure.Services.AppAuthentication.Unit.Tests/Mocks/MockProcessManager.cs

```diff
@@ -52,6 +53,9 @@ internal MockProcessManager(MockProcessManagerRequestType requestType)
                 case MockProcessManagerRequestType.ProcessNotFound:
                     throw new Exception(Constants.ProgramNotFoundError);
 
+                case MockProcessManagerRequestType.Failure:
+                    return Task.FromResult(new Tuple<bool, string>(false, tokenResult));
+
```

Comment:
Please send an error in the tokenResult, and check for that error in the test case.